game.py

```diff
@@ -0,0 +1,148 @@
+                if not flag:
+                    print("Please enter a word on one of the cards. Guess again\n")
+                if flag:
+                    break
+            system("clear")
+        else:
+            input()
+    
+    def check_guess(self):
+        '''There are 4 options.
+        1)It is their teams word. 2)It is the other team's word 
+        3)It is a nuetral word 4)It is the death card and the game is over.'''
+        for card in self.board.cards:
+            if self.guess == card.word:
+                card.flipped = True
+                if self.team == card.color:
+                    self.guess_value = 1
+                elif self.team != card.color and card.color != "yellow" and card.color != "black":
+                    self.guess_value = 2
+                elif card.color == "yellow":
+                    self.guess_value = 3
+                elif card.color == "black":
+                    self.guess_value = 4
+                    self.game_over = True
+                break
+        if self.guess_value == 1:
+            self.code_number -= 1
+        else:
+            self.code_number = 0
+
+    def check_win(self):
+        '''Function ends the game on a few conditions:
+        1) If Blue or Red pick the death card the game will end.
+        2) If either team gets their entire word set the game will end.'''
+        color = ["Blue","Red"]
+
+        if self.game_over and not self.skip:
+            print(f"{self.team} picked the death card.")
+            print("The game is over!")
+            print(f"{color[0] if self.team.capitalize() == color[1] else color[1]} wins! Congratulations!")
+
+        # This runs twice in order to check the red and the blue decks
+        for i in color:
+            all_team_color_cards = [card.word for card in self.board.cards if card.color.capitalize() == i]
+            all_flipped_team_color_cards = [card.word for card in self.board.cards if card.flipped and card.color.capitalize() == i]
+
+            if all_team_color_cards == all_flipped_team_color_cards:
+                print("The game is over!")
+                print(f"{i} Team correctly guessed all of their cards.")
+                print(f"{i} wins! Congratulations!")
+                self.game_over = True
+
+    def update_state(self):
+        '''Check if guessing is still true.
+        When guessing is no longer true, end game if game is over or update the team color.'''
+        self.guessing = (self.guess_value <= 1 and self.code_number > 0) and not self.game_over
+        if not self.guessing:
+            # This puts the game through an extra loop which allows render() to print out the non-answer board one time after the turn is over.
+            self.skip = 1 if self.skip == 0 else 0
+            # this makes so that create_code_word does not run
+            if self.skip:
+                self.guessing = True
+            if self.game_over and not self.skip:
+                sys.exit()
```

Comment:
The program point of exit should be obvious, not buried. Combine this with my comment on `skip` and you could do something like this.

```python
def main():
    ...

    while True:
        game.render()

        if game.skip:
            continue

        if game.game_over:
            break

        ...
```